core-model/src/main/java/com/sequenceiq/cloudbreak/domain/ClusterDefinition.java

```diff
@@ -152,4 +159,25 @@ public String getStackVersion() {
     public void setStackVersion(String stackVersion) {
         this.stackVersion = stackVersion;
     }
+
+    public boolean isArchived() {
+        return archived;
+    }
+
+    public Long getDeletionTimestamp() {
+        return deletionTimestamp;
+    }
+
+    public void setArchived(boolean archived) {
```

Comment:
What do you think about adding `@Override` to methods that implement declarations in interfaces? It can help avoid some errors.

Reply:
Absolutely right. I checked for other archived resources, and for some it was missing as well. Thank you!